File: client/src/app/api/models/order-vm.ts
/* tslint:disable */
import { ObjectID } from "./object-id";
import { ProductVm } from "./product-vm";
import { UserVM } from "./user-vm";

interface BasketProduct {
  product: ProductVm;
  quantity: number;
  price: number;
  totalItemPrice: number;
}
export interface OrderVm {
  createdAt?: string;
  updatedAt?: string;
  id?: string;
  user: UserVM;
  status: string;
  basket: Array<BasketProduct>;
  coupon: ObjectID;
  address: string;
  note: string;
}

File: android/FragranceCart-WishList/app/src/main/java/com/delaroystudios/fragrancecart/Interfaces/ApiOrders/OrderVm.java
package com.delaroystudios.fragrancecart.Interfaces.ApiOrders;

import com.delaroystudios.fragrancecart.Interfaces.ApiCleanersProduct.ProductVm;
import com.delaroystudios.fragrancecart.Interfaces.ApiLogin.UserVM;

/**
 * Created by hp on 12/13/2018.
 */

 class OrderProduct{
    ProductVm id;
    int quantity;
}
public class OrderVm {

    String id;
    UserVM user;
    String status;


}
